README.md

```diff
@@ -1 +1,120 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [HELLO-RAILS-BACK-END] <a name="about-project"></a>
+
+**[Hello-Rails-Back-End]** is a project created using Rails API only approach to provide random greetings in form of API to any frontend devices
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Hello-Rails-Back-End built with Rails API</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Built with Ruby on Rails**
+- **Connected to Postgresql**
+- **Configured Reck-cors for Usage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- to install Ruby
+- to install Rails
+
+The React/Redux front end link is given below:
+
+[Hello-React-Front-End](https://github.com/devalibello/hello-react-front-end)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- `git clone https://github.com/devalibello/hello-rails-back-end`
+- `cd hello-rails-back-end`
+- `code .` <!-- If you are using VS Code -->
+
+Open your terminal:
+
+Run:
+`bundle install`
+`rails s`
+
+After the server starts running, go to your browser and visit the following link:
+
+`http://localhost:3000/`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Ali Bello Imoukhuede**
+
+- GitHub: [@githubhandle](https://github.com/devalibello)
+- Twitter: [@twitterhandle](https://twitter.com/i_am_aalee)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ali-bello-imoukhuede/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [Add more endpoints]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Uthmanbello/hello-rails-back-end/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please show me support by giving it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my colleagues at Microverse for their support during the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/devalibello/hello-rails-back-end/blob/setup/LICENSE)licensed.
+
```

Comment:
- [ ] The MIT link under the **License** section in your README file is broken. Kindly make sure all the links in your README file are valid links to make your project look more professional.

Reply:
Hi @LionRouge1 
The readME license has been fixed